src/index.js

```diff
@@ -1,14 +1,12 @@
-const puppeteer = require('puppeteer');
-const fetch = require('node-fetch');
-const prompt = require('prompt');
-require('dotenv').config();
-
+const puppeteer = require('puppeteer')
+const fetch = require('node-fetch')
+const prompt = require('prompt')
+require('dotenv').config()
```

Comment:
**[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Expected 1 empty line after require statement not followed by another require. [import/newline-after-import](https://eslint.org/docs/rules/import/newline-after-import)